### ISSUE TITLE
About training with NeRF-DS dataset

### ISSUE THREAD

**Issue by EESJGong** [ingra14m/Deformable-3D-Gaussians#29]
When I was training on the  NeRF-DS cup_ Novel_ view, the PSNR at 11000 iteration is about 30, but at 12000 iteration, the PSNR mutates to 5 and the loss soars to 0.49 and remains unchanged

**Comment by ingra14m** [comment]
Generally, this is a problem of excessively high learning rates. I solved this issue in my implementation by using an exponentially decaying learning rate through the Deformation Field.

**Comment by changyq12** [comment]
可以提供一下训练的参数么，我也遇到了这样的问题

**Comment by ingra14m** [comment]
hi @changyq12, u can use the latest code for stable training.
